connector/src/main/java/dev/streamx/aem/connector/impl/AemReplicationEventHandlerConfig.java

```diff
@@ -0,0 +1,22 @@
+package dev.streamx.aem.connector.impl;
+
+import com.day.cq.commons.Externalizer;
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+import org.osgi.service.metatype.annotations.Option;
+
+@ObjectClassDefinition
+public @interface AemReplicationEventHandlerConfig {
+
+  @AttributeDefinition(
+      name = "Push From",
+      description = "Type of AEM instance (author or publish) from which "
+          + "the content should be pushed to StreamX.",
+      options = {
+          @Option(label = Externalizer.AUTHOR, value = Externalizer.AUTHOR),
```

Comment:
I think we should not use constants from Externalizer service as it's not directly related to runmodes. We can just define constants (or find better constants related to the actual run modes https://experienceleague.adobe.com/en/docs/experience-manager-65/content/implementing/deploying/configuring/configure-runmodes#defining-configuration-properties-for-a-run-mode )